lms/static/sass/discussion/views/_thread.scss

```diff
@@ -30,11 +29,25 @@ body.discussion, .discussion-module {
 
     .post-header-actions {
       @include float(right);
-      width: flex-grid(3,12);
     }
 
-    .posted-details {
-      color: $gray-d1;
+    .post-body {
+      width: flex-grid(10,12);
```

Comment:
Cool, flexbox. Should make sure whoever does the a11y review of this knows we're using it here, I'm not sure if we have any other flexbox in production and if there are a11y implications.